Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -207,15 +207,7 @@
                     };
                 }
 
-                $scope.suppliersSelectOptions = selectLazyLoading('api/organization', false, formatSupplier, ['take=100','orgId=' + user.currentOrganizationId]);
-
-                function formatSupplier(supplier) {
-                    var result = '<div>' + supplier.text + '</div>';
-                    if (supplier.cvr) {
-                        result += '<div class="small">' + supplier.cvr + '</div>';
-                    }
-                    return result;
-                }
+                $scope.suppliersSelectOptions = selectLazyLoading('api/organization', false, Kitos.Helpers.FormatHelper.formatOrganizationWithCvr, ['take=100','orgId=' + user.currentOrganizationId]);
```

Comment:
as far as I remember this formatting was also used in data processing when selecting data processors and sub data processors.. have you checked that out?

Reply:
I tried looking for it, but I couldn't found, now I got it, thanks